src/transformers/utils/import_utils.py

```diff
@@ -420,12 +420,16 @@ def is_mambapy_available():
     return False
 
 
-def is_torch_mps_available():
+def is_torch_mps_available(min_version=None):
```

Comment:
nit - it would be useful for anyone wanting to use this

```suggestion
def is_torch_mps_available(min_version: Optional[str] = None):
```

Reply:
Done 👍🏻